router.js

```diff
@@ -0,0 +1,29 @@
+
+'use strict'
+
+const express = require('express')
+const router = express.Router()
+
+// require our controllers
+const EntryController = require('./controllers/EntryController')
+
+// require our models that node-resful will transform
+var entryModel = require('./models/Entry')
+
+// Routes - Define routes here
+router.get('/entries', EntryController.get)
+
+// you can also for route that need a little amount of code,
+// declare the middleware directly here
```

Comment:
What if I wanted to declare a larger amount of middleware? How would you structure it if you it was to be in a file of its own?